package.json

```diff
@@ -36,7 +36,7 @@
     "flow-typed": "^2.5.2",
     "jest": "24.8.0",
     "react": "^16.8.3",
-    "react-native": "^0.60"
+    "react-native": "^0.63.4"
```

Comment:
Any reason this is bumped? It could cause a lot of headaches to fix CI and potentially the local build to upgrade react.

Reply:
It might have changed while setting the version for TVOS sample app. I am going to revert.